dex/src/main/scala/com/wavesplatform/dex/db/leveldb/Key.scala

```diff
@@ -10,8 +10,10 @@ trait Key[V] {
 
   override lazy val toString: String = BigInt(keyBytes).toString(16)
 
+  def canEqual(other: Any): Boolean = other.isInstanceOf[Key[V]]
+
   override def equals(obj: scala.Any): Boolean = obj match {
-    case that: Key[V] => java.util.Arrays.equals(this.keyBytes, that.keyBytes)
+    case that: Key[V] => that.canEqual(this) && java.util.Arrays.equals(this.keyBytes, that.keyBytes) && name == that.name
```

Comment:
We don't need to compare `name`, it's just a label. A Key is identified by `keyBytes`

Reply:
fixed